spec/models/organization_spec.rb

```diff
@@ -21,8 +23,39 @@
     end
   end
 
+  context 'scopes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.